packages/react/src/components/Modal/Modal.tsx

```diff
@@ -12,18 +12,23 @@ export interface ModalProps {
   style?: CSSProperties,
 }
 
-const Modal = ({ title = '', size = 'default', onClose, footerSlot, className, style, children }: ModalProps) => {
+export type Ref = HTMLDivElement;
```

Comment:
we should avoid having exports with the same name for easier usage, 
if we have to export this, prefer `ModalRef` or something similar